packages/react-components/react-carousel-preview/library/src/components/createCarouselStore.ts

```diff
@@ -3,15 +3,31 @@ import { type CarouselStore } from './CarouselContext.types';
 export const createCarouselStore = (initialValue: string | null): CarouselStore => {
   let values: string[] = [];
   let activeValue: string | null = initialValue;
+  let navDirection: 'prev' | 'next' = 'next';
+  let loopCount = 0;
 
   let listeners: Array<() => void> = [];
 
   const carouselStore = {
-    setActiveValue(newValue: string | null) {
+    setActiveValue(newValue: string | null, circular?: boolean) {
+      if (activeValue && newValue && activeValue !== newValue) {
```

Comment:
Is `''` (empty string) a valid value?

Reply:
Hmm good question - I can't see a need for it, but it technically wouldn't be invalid as that would still match a card that had value of empty string if it existed.